rllib/offline/json_writer.py

```diff
@@ -94,15 +96,16 @@ def _get_file(self):
         return self.cur_file
 
 
-def _to_jsonable(v, compress):
+def _to_jsonable(v, compress: bool) -> Any:
     if compress and compression_supported():
         return str(pack(v))
     elif isinstance(v, np.ndarray):
         return v.tolist()
     return v
 
 
-def _to_json(batch, compress_columns):
+def _to_json(batch: SampleBatchType,
+             compress_columns: List[str]) -> SerializedType:
```

Comment:
```suggestion
             compress_columns: List[str]) -> str:
```

Please use `str` here and elsewhere.